keel/api/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.conf.urls import url, include
 from django.urls import path
 
-from .v1.auth.router import urlpatterns as auth_url
+# from .v1.auth.router import urlpatterns as auth_url
+
+from .views import CustomerLeadView
```

Comment:
implementation is correct, but can you move the lead related api files[urls, views and serializers] to leads folder itself in the api folder.
and then import that urls file and include those urls as whole...
as already implemented for auth, which is commented on line4
Because we would have further APIs too and we want to segregate them on app basis.

Reply:
I'm not sure I understand this, Should I create a new leads folder in the api folder also?

Reply:
yes

Reply:
Updated, please check